File: 3rd Semester/Advanced Programming Methods(Done)/Laboratory/C#LabAssignment/sharplab/Model/Utils/MyList.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace sharplab.Model.Utils
{
    class MyList<E>:IMyList<E>
    {
        private IList<E> list;
        public MyList()
        {
            list = new List<E>();
        }
        public void Add(E el)
        {
            list.Add(el);
        }

        public override string ToString()
        {
            String msg = "";
            foreach (E el in list)
                msg += el.ToString() + ";";
            return msg;
        }
    }
}

File: 3rd Semester/Advanced Programming Methods(Done)/Laboratory/src (BUX)/Model/Utils/MyList.java
package Model.Utils;
import java.util.*;

public class MyList<E> implements MyIList<E>{
    private List<E> list;
    public MyList() {
        list = new ArrayList<E>();
    }
    public void add(E el){
        list.add(el);
    }

    public String toString(){
        String msg="";
        for (E el:list)
            msg+=el.toString()+";";
        return msg;
    }

    public List<E> getContent(){return list;}

    public void print() {
        for(E l:list)
        {
            String str = l.toString();
            System.out.println(str);
        }
    }

}